src/Compilers/Extension/CompilerExtension.csproj

```diff
@@ -24,17 +21,15 @@
     <StartProgram>$(DevEnvDir)devenv.exe</StartProgram>
     <StartArguments>/rootsuffix RoslynDev /log</StartArguments>
     <VSSDKTargetPlatformRegRootSuffix>RoslynDev</VSSDKTargetPlatformRegRootSuffix>
-    <MinimumVisualStudioVersion>$(VisualStudioVersion)</MinimumVisualStudioVersion>
```

Comment:
The property is set to this same value in the Settings.props of this repo - hence the tool removed it as unnecessary.